server/src/test/java/org/elasticsearch/action/admin/cluster/configuration/AddVotingConfigExclusionsRequestTests.java

```diff
@@ -131,15 +322,8 @@ public void testResolveAndCheckMaximum() {
                 .coordinationMetaData(CoordinationMetaData.builder().addVotingConfigExclusion(otherNode1Exclusion).build()));
         final ClusterState clusterState = builder.build();
 
-        assertThat(makeRequest().resolveVotingConfigExclusionsAndCheckMaximum(clusterState, 3, "setting.name"),
```

Comment:
Did you mean to remove these assertions? Can you explain why?

Reply:
This was due to `makeRequest()` will now throw 

`IllegalArgumentException: Please set node identifiers correctly. One and only one of [node_name], [node_names] and [node_ids] has to be set`

because it's not passing in any node identifiers, as the definition of `makeRequest` is 

```
    private static AddVotingConfigExclusionsRequest makeRequest(String... descriptions) {
        return new AddVotingConfigExclusionsRequest(descriptions);
    }
```

So the new check added inside `AddVotingConfigExclusionsRequest` constructor throws exception under this scenario.

In addition, the remaining assertions using `makeRequest("_local")` seems to verify the logic `AddVotingConfigExclusionsRequest#resolveVotingConfigExclusionsAndCheckMaximum` already.

Do you think I should still add back these assertions with tweak for testing `AddVotingConfigExclusionsRequest#resolveVotingConfigExclusionsAndCheckMaximum`  more thoroughly?

Reply:
Looks ok now, I will re-review rather than trying to trace the history here.